src/nanoemoji/svg.py

```diff
@@ -304,6 +306,9 @@ def _add_glyph(svg: SVG, color_glyph: ColorGlyph, reuse_cache: ReuseCache):
     upem_to_vbox = vbox_to_upem.inverse()
 
     # copy the shapes into our svg
+    XLINK_NAMESPACE = "http://www.w3.org/1999/xlink"
```

Comment:
you can `from picosvg.svg_meta import xlinkns`, then use as `xlinkns()`

Reply:
Done